components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/impl/BasicAuthAuthenticator.java

```diff
@@ -77,10 +77,7 @@ public boolean authenticate(Request request, Response responder, ServiceMethodIn
 
     private boolean authenticate(String username, String password) {
         //todo improve
-        if (username.equals(password)) {
-            return true;
-        }
-        return false;
+        return username.equals(password);
```

Comment:
What's this? Do we have an idea to implement this? If so please create relevant GitHub issue.